spec/wrappers/wakes/google_analytics_api_wrapper_spec.rb

```diff
@@ -6,18 +6,16 @@
   describe '#get_page_of_pageviews' do
     let(:rows) { [['/path', 151], ['/path/2', 500]] }
     let(:results) { double('Results', :rows => rows, :items_per_page => 2) }
-    let(:analytics_service) do
-      instance_double(Google::Apis::AnalyticsV3::AnalyticsService, :get_datum_ga => results)
```

Comment:
@chrismaddocks can we preserve the usage of `instance_double` here? That causes the spec to fail if the method isn't available anymore. If we didn't have that here, you wouldn't have known to change this again!